superset-frontend/src/dashboard/components/nativeFilters/FilterBar/utils.ts

```diff
@@ -18,7 +18,52 @@
  */
 
 import { areObjectsEqual } from 'src/reduxUtils';
-import { DataMaskStateWithId, Filter, FilterState } from '@superset-ui/core';
+import {
+  DataMask,
+  DataMaskStateWithId,
+  Divider,
+  Filter,
+  FilterState,
+} from '@superset-ui/core';
+import { testWithId } from 'src/utils/testUtils';
+import { FilterBarLocation } from 'src/dashboard/types';
+
+interface CommonFiltersBarProps {
```

Comment:
Can we leave only utility functions in this file and move type definitions to a types file?